neurodamus/utils/memory.py

```diff
@@ -256,6 +256,7 @@ def __init__(self) -> None:
         self.metype_counts = Counter()
         self.synapse_counts = Counter()
         self.suggested_nodes = 0
+        self.sim_estimate_factor = 2.5
```

Comment:
This could also be an all caps file wide variable, with a comment on why 2.5 and how it's going to bu used :)

Reply:
Fair enough! Fill fix, thanks!